File: gzoom2-fe/app/src/app/api/model/uom.ts
import { UomType } from '../../view/ctx-ac/uom-type/uom_type';
import { VariableGridArray } from "app/layout/tables/table-editing-cell/table-editing-cell-configuration";
import { UomRatingScale } from './uomRatingScale';

/**
 * Model of a Uom.
 */
export class Uom {

  uomRatingScale?: UomRatingScale;
  constructor(
    public uomId?: string,
    public uomTypeId?: string,
    public uomType?: string,
    public abbreviation?: string,
    public description?: string,
    public descriptionLang?: string,
    public abbreviationLang?: string,
    public decimalScale?: number,
    public minValue?: number,
    public maxValue?: number,
    public variableGridArray?: VariableGridArray) { }
}

File: gzoom2-be/goalfile-repository/src/main/java/it/mapsgroup/gzoom/entity/uom/Uom.java
package it.mapsgroup.gzoom.entity.uom;

import lombok.*;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Uom {

    private String id;
    private UomType type;
    private String abbreviation;
    private String abbreviationLang;
    private String description;
    private String descriptionLang;
    private Integer decimalScale;
    private Double minValue;
    private Double maxValue;
}